src/Sylius/Bundle/ProductBundle/DependencyInjection/SyliusProductExtension.php

```diff
@@ -57,20 +65,28 @@ private function prependAttribute(ContainerBuilder $container, array $config)
         }
 
         $container->prependExtensionConfig('sylius_attribute', array(
-                'classes' => array(
+                'resources' => array(
                     'product' => array(
-                        'subject'         => $config['classes']['product']['model'],
+                        'subject'         => $config['resources']['product']['classes']['model'],
                         'attribute'       => array(
-                            'model'       => 'Sylius\Component\Product\Model\Attribute',
-                            'repository'  => 'Sylius\Bundle\TranslationBundle\Doctrine\ORM\TranslatableResourceRepository',
-                            'translation' => array(
-                                'model' => 'Sylius\Component\Product\Model\AttributeTranslation'
+                            'classes' => array(
+                                'model'       => 'Sylius\Component\Product\Model\Attribute',
```

Comment:
You're not using `X::class` here compared to everywhere else, is there a reason?